src/pages/overlay/index.scss

```diff
@@ -1,10 +1,40 @@
 @import '../../variables.scss';
 
+:root{
+    //used to scale various elements based on available height
+    //but clamped between 6px and 10px to prevent tiny/giant UI
+    //the values are chosen so that in a normal desktop resolution
+    //1 base unit === 10px to make the math easier when using it
+    //to scale elements.
+    //
+    //usage example:
+    //  width: calc(5 * var(--alveus-base-size));
+    //  => which would be 50px in normal views and
+    //       would scale down to 30px
+    --alveus-base-size: clamp(6px, 1.46vh, 10px);
```

Comment:
Curious why this is being defined as a CSS var, rather than a SCSS var? Maybe personal pref, but with variables already defined in SCSS, I'd probably lean on a consistent approach to them?

Reply:
Good catch. I tried to use media queries first to make the css variable dynamic (in steps), but moved to the clamp solution without media queries. I will update the PR later.